lib/postprocessing/image.js

```diff
@@ -178,6 +181,35 @@ async function needsImagePostProcess(rendition) {
     return false;
 }
 
+async function processWatermark(img, renditionInstructions, outdir) {
+    console.log('Processing watermark...');
+
+    const watermarkProcessor = new WatermarkProcessor(renditionInstructions);
+    const resizedWatermarkPath = renditionInstructions.watermark.name ? path.join(outdir, 'resized-'+ renditionInstructions.watermark.name)
+        : path.join(outdir, 'resized-'+ path.basename(renditionInstructions.watermark.path));
```

Comment:
this is a long/confusing line of code. maybe change it to an if statement so its more readable?